docs/REST API/rest-api.md

```diff
@@ -5585,6 +5586,48 @@ Resends a configuration profile for the specified host.
 
 `Status: 202`
 
+### Batch-modify custom OS settings(configuration profiles)
```

Comment:
TODO: @noahtalerman: Error message if MDM is turned off. Think about cases:
- MDM is off for Windows and Apple
- MDM is off for just one

Also update the test plan